### PR TITLE
Adding test cases for Pipeline run details page as part of PAC

### PR DESCRIPTION
**Pull request by VeereshAradhya**
Added test cases for Pipelines As Code along with test data

Jira Id: [ODC-5149](https://issues.redhat.com/browse/ODC-5149)

Acceptance Criteria: 

1. As a user, looking at the Pipelines page in the Developer Console, I should be able to see a list of (a) Git repositories that are added to the namespace for PAC execution AND (b) all pipelines in the namespace
2. As a user, I should be able to navigate to a details page of the git repo.
    1. This details page should provide access to (a) details of the git repo and (b) a list of pipeline runs.
    2. This PLR tab should show additional information than the typical PLR List view, including SHA (commit id), commit message, branch & trigger type
3. As a user, when looking at a Pipeline Run Details page, if associate with a git repo (PAC),
4. Indicate that it's from a specific git repo rather than a PL resource
5. Include the SHA (commit id), commit message, branch & trigger type

Pre-conditions for Epic validation:

1. Openshift Pipelines is installed 
2. Pipelines As Code is installed

Checks required for approving Epic gherkin scripts PR:

- [x] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [x] Add @to-do for automation possible scenarios
- [x] Add @regression or @smoke based on the importance of the scenario
- [ ]  Update the test scenarios count in Automation status confluence Report
- [x]  Check for the linter issues by executing yarn run gherkin-lint on frontend folder [Skip epic number tags related linter issues]

@makambalaji @sanketpathak  @karthi please review

### CODE REVIEW

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature [hunk 1]
```diff
@@ -101,7 +101,22 @@ Feature: Perform Actions on repository
...
 
         @regression @to-do
-        Scenario Outline: Delete the repository from the Repository details page: P-11-TC09
+        Scenario Outline: Pipeline Run Details page for the repository: P-11-TC09
+            Given pipeline run is displayed for "<repository_name>"
+              And user is at the repositories page
+             When user clicks Last Run value of repository "<repository_name>"
+             Then user will be redirected to Pipeline Run Details page
+              And user is able to see Details, YAML, TaskRuns, Logs and Events tabs
+              And Details tab is displayed with field names Name, Namespace, Labels, Annotations, Created At, Owner, Status, Pipeline, Repository, Branch, commit id and Event type
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature:110]: For a pipelineRun, I think only these four fields (Repository, Branch, commit id and Event type) are related to Repository info and rest is common for any resource.

Suggestions:
1. Update the scenario to look for these four fields.
2. commit Id will be a external link, and on hover you will get the tooltip of the commit message. Shall we include this ?

**Review by karthikjeeyar — COMMENTED** [review]
If PAC is installed,  `Repositories`  tab will be listed in Admin -> pipelines as well. Do we include that scenario ?.

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature [hunk 2]
```diff
@@ -101,7 +101,33 @@ Feature: Perform Actions on repository
...
+              And Details tab is displayed with fields Repository, Branch, Commit id and Event type
+              And Actions dropdown display on the top right corner of the page
+
+        Examples:
+                  | repository_name |
+                  | test-repo       |
+
+
+        @regression @to-do
+        Scenario Outline: Pipeline Runs tab of the Reposiotry details page: P-11-TC10
```
Comment (karthikjeeyar) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature:119]: nit: typo
```suggestion
        Scenario Outline: Pipeline Runs tab of the Repository details page: P-11-TC10
```

**Review by karthikjeeyar — APPROVED** [review]
/lgtm

**Comment by makambalaji** [comment]
This pr doesn't have any changes related to dev code and docs. For merging purpose, adding below labels
/label px-approved
/label qe-approved
/label docs-approved

**Comment by makambalaji** [comment]
/hold

There is one accessibility issue in helm chart in before hook. so fixed it in #9727

**Comment by makambalaji** [comment]
/unhold
PR: #9727 is in merge pool

**Comment by makambalaji** [comment]
/unhold
PR: #9727 is in merge pool

**Pull request merged**